packages/server/test/integration/clients.test.ts

```diff
@@ -13,7 +13,7 @@ describe('Clients', () => {
   beforeAll(async () => {
     await setupApp()
     clients = app.clients
-    querySpy = jest.spyOn(clients, 'query')
+    querySpy = jest.spyOn(clients as any, 'query')
```

Comment:
PP: Cleaner IMHO

```suggestion
    querySpy = jest.spyOn<any>(clients, 'query')
```

Reply:
You need more than one type argument to make this work. I that case I'd argue writing as any is less anoyying than writing jest.spyOn<any, any>(clients, 'query').